src/index.ts

```diff
@@ -71,7 +69,10 @@ export function getParams() {
     head = `${owner}:${head}`
   }
 
-  const labels = core.getInput("labels").split(",").map((l) => { return l.trim() });
+  let labels;
```

Comment:
```suggestion
  let labels:string[] =[];
```
Could we return an empty array instead of undefined?